src/views/ModelPlan/TaskList/Basics/index.tsx

```diff
@@ -1,29 +1,480 @@
+                    </FieldGroup>
+                    {values.cmsCenters.includes('CMMI') && (
+                      <FieldGroup
+                        error={!!flatErrors.cmmiGroup}
+                        className="margin-top-4"
+                      >
+                        <FieldArray
+                          name="cmmiGroup"
+                          render={arrayHelpers => (
+                            <>
+                              <legend className="usa-label text-normal">
+                                {t('cmmiGroup')}
+                              </legend>
+                              <FieldErrorMsg>
+                                {flatErrors.cmmiGroup}
+                              </FieldErrorMsg>
+
+                              {(t('cmmiGroups', {
+                                returnObjects: true
+                              }) as string[]).map((item, key) => {
+                                return (
+                                  <Fragment key={item}>
+                                    <Field
+                                      as={CheckboxField}
+                                      id={`new-plan-cmmiGroup--${key}`}
+                                      name="cmmiGroup"
+                                      label={item}
+                                      value={translateCmmiGroup(item)}
+                                      checked={values.cmmiGroup.includes(
+                                        translateCmmiGroup(item)
+                                      )}
+                                      onChange={(
+                                        e: React.ChangeEvent<HTMLInputElement>
+                                      ) => {
+                                        if (e.target.checked) {
+                                          arrayHelpers.push(e.target.value);
+                                        } else {
+                                          const idx = values.cmmiGroup.indexOf(
+                                            e.target.value
+                                          );
+                                          arrayHelpers.remove(idx);
+                                        }
+                                      }}
+                                    />
+                                  </Fragment>
+                                );
+                              })}
+                            </>
+                          )}
+                        />
+                      </FieldGroup>
+                    )}
+
+                    <div className="margin-top-6 margin-bottom-3">
+                      <Button
+                        type="submit"
+                        disabled={!(dirty || isValid)}
+                        onClick={() => setErrors({})}
+                      >
+                        {h('next')}
+                      </Button>
+                    </div>
+                    <Button
```

Comment:
I think these two save methods are too complex for what they need to be.  Both can be the normal submit functionality, except for the autosave.  I think it makes most sense to not autosave at all of the modelName is empty.  If** we do want to autosave, we can handle that in the submit handler function.